src/Files/Strings/en-US/Resources.resw

```diff
@@ -2705,4 +2705,10 @@ We use App Center to track which settings are being used, find bugs, and fix cra
   <data name="RunWithPowerShell" xml:space="preserve">
     <value>Run with PowerShell</value>
   </data>
-</root>
\ No newline at end of file
+  <data name="SetFilesAsDefaultWarning" xml:space="preserve">
+    <value>Do not remove Files from your device without turning off this option. Doing so will prevent you from opening File Explorer.</value>
+  </data>
+  <data name="ShowFolderSizesWarning" xml:space="preserve">
+    <value>Calculating folder sizes is resource intensive and may cause your CPU usage to increase</value>
```

Comment:
'.' at the end

Reply:
Can you add these as review suggestions so I can commit them?